Cargo.toml

```diff
@@ -29,9 +30,10 @@ rustversion = "1.0.7"
 
 [features]
 default = ["pretty_errors", "module_disambiguation"]
-pretty_errors = ["proc-macro-error"]
+pretty_errors = ["dep:proc-macro2-diagnostics", "dep:proc-macro2"]
 module_disambiguation = ["heck"]
 fail-on-warnings = [] # Forces compilation to fail if any warnings are given. Used in CI.
+proc-macro-error = [] # This feature is here for backwards-compatibility with the implicit feature from the no longer used `proc-macro-error` dependency.
```

Comment:
No need for backwards compatibility since we already have other breaking changes.
```suggestion
```